pkgs/development/python-modules/vulkan/default.nix

```
@@ -0,0 +1,67 @@
+{
+  buildPythonPackage,
+  cffi,
+  fetchFromGitHub,
+  gitUpdater,
+  inflection,
+  jinja2,
+  lib,
+  pysdl2,
+  setuptools,
+  substituteAll,
+  vulkan-loader,
+  wheel,
+  xmltodict,
+}:
+
+buildPythonPackage rec {
+  pname = "vulkan";
+  version = "1.3.275.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "realitix";
+    repo = "vulkan";
+    rev = version;
+    hash = "sha256-b1jHNKdHF7pIC6H4O2yxy36Ppb60J0uN2P0WaCw51Gc=";
+  };
+
+  patches = [
+    # OSError: Cannot find Vulkan SDK version. Please ensure that it is installed and that the <sdk_root>/<version>/lib/ folder is in the library path
+    (substituteAll {
+      src = ./vulkanapi-paths.patch;
+      vulkan_loader = vulkan-loader;
+    })
+  ];
+
+  passthru.updateScript = gitUpdater { };
+
+  buildInputs = [
+    vulkan-loader
+  ];
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  dependencies = [
+    inflection
+    jinja2
+    pysdl2
+    xmltodict
+    cffi
+  ];
+
+  pythonImportsCheck = [
+    "vulkan"
+  ];
+
+  meta = {
+    description = "The ultimate Python binding for Vulkan API";
```

```suggestion
    description = "Ultimate Python binding for Vulkan API";
```
See https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes